package.json

```diff
@@ -23,16 +23,16 @@
   },
   "private": true,
   "dependencies": {
```

Comment:
some of (dev)dependencies should be upgraded, create a new app using the angular cli and compare

Reply:
I think I got them all, I ignore the ckeditor ones(since they are going away) and eslint/formly ones(since I don't know enough about them).